src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api/Controllers/ProviderLocationsGetController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using MediatR;
```

Comment:
The naming of this controller seems odd.  Wouldn't GetProviderLocationsController be better (and I know it was previously called that, I don't understand the name change)

Reply:
I agree the name does not sound right. Just avoid using GET in the name, for ex: ProviderCourseLocationsController